### PR TITLE
test(elvis): spaces around |, no spaces after ( and before )

### PR DESCRIPTION
**Pull request by z8674558**

### CODE REVIEW

**Comment by z8674558** [comment]
@zmstone trying out new rules here. Once you approve, I will apply this change on emqx.git.

**Comment by zmstone** [comment]
can we add `|` and `||` here:
https://github.com/inaka/elvis_core/blob/c4ffeb071ec7c01e5229091f32ce2e091f2cfbcf/src/elvis_style.erl#L323

maybe even a PR to upstream

**Comment by zmstone** [comment]
these are the defaults.
https://github.com/inaka/elvis_core/blob/c4ffeb071ec7c01e5229091f32ce2e091f2cfbcf/src/elvis_style.erl#L145-L150

 maybe we should enable all in our elvis.config

**Comment by z8674558** [comment]
https://github.com/inaka/katana-code/blob/7e635609446b6f1e4cdd233f6dcb1d5ab34d99b8/src/ktn_code.erl#L25
This is the all types of tokens we can get from an `.erl` file.

**Comment by z8674558** [comment]
Some of operators are packed to `op` map. `|` and `||` are not.

https://github.com/inaka/katana-code/blob/7e635609446b6f1e4cdd233f6dcb1d5ab34d99b8/src/ktn_code.erl#L627

File: scripts/elvis_more_rules.erl [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    throw({invalid_file, File}).
+
+find_encoding(Content) ->
+    case epp:read_encoding_from_binary(Content) of
+        none -> utf8;
+        Enc  -> Enc
+    end.
+
+
+% eunit
```
Comment (k32) [inline scripts/elvis_more_rules.erl:99]: Perhaps wrap tests in `-ifdef(TEST).`?

Comment (z8674558) [reply scripts/elvis_more_rules.erl]: ty, I was not aware of it.

**Comment by zmstone** [comment]
> Some of operators are packed to `op` map. `|` and `||` are not.
> 
> https://github.com/inaka/katana-code/blob/7e635609446b6f1e4cdd233f6dcb1d5ab34d99b8/src/ktn_code.erl#L627

I believe `|` is `cons`.
https://github.com/inaka/katana-code/blob/7e635609446b6f1e4cdd233f6dcb1d5ab34d99b8/src/ktn_code.erl#L36

**Comment by zmstone** [comment]
Maybe move this module to our elvis fork ?
so we can reuse without having to copy the module to another project ?
https://github.com/emqx/elvis

**Comment by z8674558** [comment]
> Maybe move this module to our elvis fork ?

sure. by doing so we should be write this more simple.

**Comment by zmstone** [comment]
this PR can be reopened, we still need the config changes and HOCON code changes ?

**Pull request closed without merging**